.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,47 @@
+name: docker_release
+
+on:
+  push:
+    branches: [ dev ]
+    paths-ignore:
+      - 'docs_src/**'
+      - 'README.md'
+      - 'CHANGELOG.md'
+      - 'CITATION'
+      - 'book.toml'
+      - 'CONTRIBUTING.md'
+    tags: [ 'v*.*.*' ]
+
+jobs:
+  build_and_push:
+    name: build_and_push_docker_image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Docker metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            hubblo/scaphandre
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
These need to be added as secrets in the GitHub repo.

Reply:
@bpetit, we need someone with admin access to the repo to configure CI vars.